src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
```

Comment:
I recommend you to use declarative names  instead of single letters, in this small function it's readable, but in a bigger one you might get confused... Instead of "p", how about using "person"?
Another tip, in this case you shouldn't use parentheses because it is a single parameter in a function with no side effects. 

Example: ...(person => person.sex === 'm' && Math.ceil(person.died / 100 === century)

[Click here for more information](https://mate-academy.github.io/style-guides/javascript.html#47-use-return-statements-in-array-method-callbacks-its-ok-to-omit-the-return-if-the-function-body-consists-of-a-single-statement-returning-an-expression-without-side-effects-following-82)